src/index.js

```diff
@@ -349,7 +362,25 @@ class MiniCssExtractPlugin {
                     'reject(err);',
                   ]),
                   '};',
-                  'linkTag.href = fullhref;',
+                  '} else {',
+                  'var errorTimeout = 60000;',
```

Comment:
What about is loading more than `60000` ?

Reply:
It's a timeout threshold for css loading. To make sure chunk is resolved, I think we need a stop time to clear setInterval when the css loaded fail.